resources/js/components/Committee/Committee.vue

```diff
@@ -0,0 +1,57 @@
+<script>
+import CommitteeInfo from './CommitteeInfo';
+import CommitteeMemberList from './CommitteeMemberList';
+import CommitteeSessions from './CommitteeSessions';
+export default {
+    name: 'Committee',
+    props: {
+        info: {
+            type: String,
+            default: 'info'
+        },
+        members: {
+            type: Array,
+            default: () => []
+        },
+        meetings: {
+            type: Array,
+            default: () => []
+        }
+    },
+    components: {
+        CommitteeInfo,
+        CommitteeMemberList,
+        CommitteeSessions
+    },
+    data() {
+        return {
+            activeTab: 'mitglieder',
+            tabs: ['aufgaben', 'mitglieder', 'sitzungen']
+        };
+    },
+};
+</script>
+
+<template>
+    <div class="">
+        <nav class="ris-committee-navigation ris-without-padding-mob">
+            <ul class="ris-ul ris-committee-navigation__list">
+                <li class="ris-committee-navigation__item" v-for="tab in tabs" :key="tab">
+                    <button
+                        @click="activeTab = tab"
+                        class="ris-committee-navigation__button"
+                        :class="[`ris-committee-navigation__button`, activeTab === tab ? 'active' : '' ]">
+                        <span v-if="tab === 'sitzungen'" class="ris-i ris-i_calendar-empty" />
```

Comment:
these 3 `<span />` could be replaced with a single one, since the only difference between them is their classes and to get the actual value of the class you can create a method:
```html
<span class="ris-i" :class="getClassForTab(tab)" />
```

```js
methods: {
    getClassForTab(tab) {
        // if statements here
        return ...
    }
}
```